### PR TITLE
fix a few typographic errors in test utils file

### PR DESCRIPTION
**Pull request by logicalhan**
**What this PR does / why we need it**: 

I was running some unit tests, noticed a typo and went ahead and audited the file. 

**Which issue(s) this PR fixes**: N/A

**Special notes for your reviewer**: N/A

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Review by cheftako — COMMENTED** [review]
/ok-to-test

File: pkg/controller/testutil/test_utils.go [hunk 1]
```diff
@@ -304,7 +304,7 @@ func (m *FakeNodeHandler) Patch(name string, pt types.PatchType, data []byte, su
 	}
 	var originalNode v1.Node
 	if err = json.Unmarshal(originalObjJS, &originalNode); err != nil {
-		glog.Errorf("Failed to unmarshall original object: %v", err)
```
[the remark this answers is not in the document]
Comment (logicalhan) [reply pkg/controller/testutil/test_utils.go]: I've updated my commit to include that fix. Thanks for the extra eyes.

**Review by logicalhan — COMMENTED** [review]
Updated to include the typo that Walter pointed out.

**Review by cheftako — APPROVED** [review]
/lgtm. Thanks for fixing this.

**Comment by nikhita** [comment]
> went ahead and audited the file.

@logicalhan Thanks for taking care of all loose ends!

/retest
/sig api-machinery
/kind cleanup

Re-applying @cheftako's lgtm, but without the trailing text :)
/lgtm 

/assign @deads2k 
for approval

**Pull request merged**